src/main/java/step3/common/Game.java

```diff
@@ -0,0 +1,7 @@
+package step3.common;
```

Comment:
```suggestion
package racinggame.common;
```

단계로 구분할 경우 4단계 구현할 때 패키지 명을 바꿔야 한다.
racinggame과 같이 미션 이름을 사용하는 것은 어떨까?